test/cljs/athenaeum_web/events/login_page_test.cljs

```diff
@@ -0,0 +1,43 @@
+(ns athenaeum-web.events.login-page-test
+  (:require [cljs.test :refer-macros [deftest testing is run-tests]]
+            [day8.re-frame.test :as rf-test]
+            [re-frame.core :as rf]
+            [re-frame.db :as rf-db]
+            [athenaeum-web.test-utils :as tu]
+            [athenaeum-web.login-page.events :as login-page-events]
+            [athenaeum-web.app.subscriptions :as subs]
+            [athenaeum-web.app.effects :as effects]))
+
+(deftest login-page-navigated-test
+  (testing "When login page is navigated and user is logged in, home page is set"
+    (rf-test/run-test-sync
+     (let [navigate-to-params (tu/stub-effect ::effects/navigate-to)]
+       (tu/initialize-db)
+       (swap! rf-db/app-db assoc :login-state :logged-in)
+       (rf/dispatch [::login-page-events/login-page-navigated])
+       (is (= ["/"] @navigate-to-params)))))
+
+  (testing "When login page is navigated and user is logged in, nothing happens"
```

Comment:
I wasn't able to follow this test. Can you explain what is being tested here?
Also the docstring says `logged in`, but the test is setting the state to `:logged-out`.

Reply:
that's supposed to say logged *out. should i just remove it. i added it because the test only said what happens when the user is logged in, but it's unnecessary otherwise.